src/scalper.ts

```diff
@@ -286,7 +286,7 @@ export class Scalper {
           console.log(this.symbol, "Delta Hedge Complete");
           return;
         }
-        await sleepTime(fillScan);
+        await sleepRandom(fillScan);
```

Comment:
Does this random need to be the same as the overall period?

Reply:
Not necessarily. Instead of checking each second we just check each random 1 second (0.975 sec to 1.025 sec) x 15 which itself makes the twapInterval a 15 sec timer with 0.75 secs of net randomness